modules/varnish/manifests/monitoring.pp

```diff
@@ -39,12 +39,14 @@
       'query' => "host:${hostname_prefix[0]}* AND @fields.status:[500 TO 599]",
       'from'  => '4h',
     }
+    $action_url = "kibana5_url(${kibana_url}, ${kibana_search})"
     } else {
     $kibana_url = "https://kibana.${app_domain}"
     $kibana_search = {
       'query' => "@source_host:${hostname_prefix[0]}* AND status:[500 TO 599]",
       'from'  => '4h',
     }
+    $action_url = "kibana5_url(${kibana_url}, ${kibana_search})"
```

Comment:
Did you mean this to be `kibana3_url`, and the LogIt one only to be `kibana5_url`?

Reply:
Ugh, yes. Time to go home 😭